includes/class-wc-bfx-pay-gateway.php

```diff
@@ -521,24 +520,6 @@ public function webhook()
         exit();
     }
 
-    /**
-     * Change payment complete order status to completed for Вitfinex payments method orders.
-     *
-     * @param string         $status   current order status
-     * @param int            $order_id order ID
-     * @param WC_Order|false $order    order object
-     *
-     * @return string
-     */
-    public function change_payment_complete_order_status($status, $order_id = 0, $order = false)
```

Comment:
@dattran92 I understand removal of this to avoid override, but I'm confused how webhook will be linked in this case? it will always complete payment, how this will affect order status?

Reply:
additionally on webhook we have this issue:
```
    public function webhook()
    {
        if (!isset($_SERVER['REQUEST_METHOD']) || 'POST' !== $_SERVER['REQUEST_METHOD']) {
            return;
        }
        $payload = file_get_contents('php://input');
        $data = json_decode($payload, true);
        $order = wc_get_order($data['orderId']);
        if ('COMPLETED' !== $data['status']) { <--- status could be PENDING here
            $order->update_status('failed');
            ret
```

Reply:
I'm not sure about the other status. Do we trigger webhook for other statuses? If yes, I can fix it. Do you want to fix in another PR? We also have that problem in the cron schedule which I'm planning to fix as well.

Reply:
I think we can fix all status related issues in this PR

Reply:
Updated! Could you check please?